lib/service/commands/info.rb

```diff
@@ -27,16 +27,45 @@
 require_relative '../command'
 require_relative '../type'
 
+require 'tty-markdown'
+require 'erb'
+require 'ostruct'
+
 module Service
   module Commands
     class Info < Command
+      TEMPLATE = <<~ERB
+      # <%= title %>
+
+      <%= text %>
+
+      <% values.each do |opts| -%>
+      ## <%= opts['label'] %>
+      *Key:* <%= opts['key'] %>
+      *Default:* <%= opts['value'] %>
+      *Value:* <%= data[opts['key']] %>
+
+      <% end -%>
+      ERB
```

Comment:
The output used here contains ruby warnings and is not easily scriptable.

```
[flight@gateway1 (ben1) ~]$ flight service info www 
/opt/flight/opt/service/vendor/ruby/2.7.0/gems/kramdown-1.16.2/lib/kramdown/converter/base.rb:102: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/opt/flight/opt/service/vendor/ruby/2.7.0/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:15: warning: The called method `initialize' is defined here
Flight environment web server

Please provide values for the following configuration items.

  HTTP Port
  Key: port
  Default: 80
  Value:

  HTTPs Port
  Key: https_port
  Default: 443
  Value:
```

I think we'd be better of using our existing table output here.

Reply:
https://github.com/openflighthpc/flight-service/pull/8/commits/d69e43548152a63a00ba1a597f9bd39e26df4193

Reply:
Looks much better.  Thanks.